.travis/install_saw.sh

```diff
@@ -32,10 +32,14 @@ mkdir -p "$DOWNLOAD_DIR"
 cd "$DOWNLOAD_DIR"
 
 #download saw binaries
-curl --retry 3 https://s3-us-west-2.amazonaws.com/s2n-public-test-dependencies/saw-0.2-2019-03-08-Ubuntu14.04-64.tar.gz --output saw.tar.gz;
+# curl --retry 3 https://s3-us-west-2.amazonaws.com/s2n-public-test-dependencies/saw-0.2-2019-03-08-Ubuntu14.04-64.tar.gz --output saw.tar.gz;
+# curl --retry 3 https://github.com/pedrotst/saw-script/releases/download/3.0.x/saw.tar.gz --output saw.tar.gz -k;
+wget --tries=10  --no-check-certificate https://github.com/pedrotst/saw-script/releases/download/3.0.x/saw.tar.gz -O saw.tar.gz
 
-mkdir -p saw && tar -xzf saw.tar.gz --strip-components=1 -C saw
```

Comment:
I would keep this commented out, for when we switch back to the "official" `saw`

Reply:
I don't like leaving this sort of comments behind since they pollute code and are easy to retrieve from git anyways.

Reply:
the only difference is `--strip-components` and this becomes useless with `wget -O` even in the future